pyquil/device.py

```diff
@@ -275,26 +275,31 @@ class Device(object):
     def __init__(self, name, raw):
         """
         :param name: name of the device
-        :param raw: raw JSON response from the server with additional information about this device
+        :param raw: raw JSON response from the server with additional information about this device.
+        Two additional keys are optionally expected in device._raw - 'isa' and 'noise_model' - which
```

Comment:
can we move this info to the class-level docstring along with the ivars? then it will render in RTD